### PR TITLE
Sample init scripts

### PR DESCRIPTION
**Pull request by captainpete**
Hi @waywardgeek. Following your suggestion from #17:

- Added `software/init_scripts` scripts for OpenRC, Upstart and systemd
- Added `software/init_scripts/README.md`
- Updated `software/README` with a sentence about the `init_scripts` folder
- Updated `software/.gitignore` to not ignore the OpenRC script

### CODE REVIEW

File: software/.gitignore [hunk 1]
```diff
@@ -1 +1 @@
-infnoise
+./infnoise
```
Comment (captainpete) [inline software/.gitignore]: does this need correction?

**Comment by manuel-domke** [comment]
Can you replace the systemd files with the ones I provided?
Maybe together with the udev rule. I'll update the docs then.

**Comment by captainpete** [comment]
np

**Comment by manuel-domke** [comment]
I've again updated the udev rule and unit files (#36). 
I'll merge the two scripts folders after _this_ pull request has been merged into master.

**Pull request merged**